app/design/adminhtml/Magento/backend/web/css/source/components/_messages.less

```diff
@@ -121,6 +121,15 @@
     }
 }
 
+#import_validation_container  {
```

Comment:
@speedy008 can you please change the selector to avoid using the ID

Reply:
Hi @sivaschenko 
I have Updated code.

Reply:
Thank you!